CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ Inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 **Note:** This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.0] - Unreleased
+### Added
+- Require Ruby > 3.1
+- Allow ActiveSupport >= 6
+### Removed
+- Support for Ruby < 3
+- Support for ActiveSupport < 6
+
```

Comment:
Since this hasn't had a major version release yet and is still in "initial development" we can add this "breaking change" as a minor version bump ([see semver rule 6](https://semver.org/spec/v1.0.0.html#spec-item-6) for details)